frontend-web/webclient/app/Grants/Editor.tsx

```diff
@@ -2331,7 +2342,7 @@ const GrantGiver: React.FunctionComponent<{
         if (!props.onTransfer) return;
 
         const result = await transferProject(
-            props.allAllocators.filter(it => !it.checked && it.id !== props.projectId)
+            props.transfers.filter(it => !it.checked && it.id !== props.projectId)
```

Comment:
Am I missing something or are `transfers` and `allAllocations` equal to each other?

Reply:
They look identical to me. I am not sure if that is intended.

Reply:
As far as I can see, these can diverge when calling `loadRevision`, so I don't think they are guaranteed to be equal.

Reply:
This is part of the initial comment, so maybe I should have read that first.